coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs

```diff
@@ -153,9 +154,23 @@ public async Task CreateDocument(IFormFile document, string userName)
                     hash = builder.ToString();
                 }
             }
-            await _dbAccess.UploadDocument(name,documentContent,hash,userName);
+            await _dbAccess.UploadDocument(name, documentContent, hash, userName);
         }
         public Task CreateCustodianWalletAsync(WalletInformation information) =>
             _custodianService.CreateWallet(information.bpn, information.name);
+
+        public async Task<bool> SubmitRegistrationAsync(string userEmail)
+        {
+            var password = new Password().Next();
```

Comment:
Why is the password needed? I don't think that is need in the nextsteps template.

Reply:
password implementation removed